src/components/Dialog-Window/Tabs/DialogTabs.tsx

```diff
@@ -40,32 +40,32 @@ const defaultTabValue = (tabContents: DialogContent): string => {
 };
 
 const tabLabelItems = (
-  tabContents: DialogContent,
+  tabContents: DialogContent | undefined,
   translation: Translation,
 ): JSX.Element[] => {
   const items = [];
-  tabContents.text
+  tabContents?.text
```

Comment:
Skal også vises dersom det er lagt til bilde eller beskrivelse

```suggestion
  const showTextTab = tabContents?.text || topicImage || description;
  showTextTab
```

Reply:
✨